Zend/zend_compile.c

```diff
@@ -6830,11 +6830,6 @@ void zend_compile_use(zend_ast *ast) /* {{{ */
 				new_name = zend_string_copy(old_name);
 
 				if (!current_ns) {
-					if (type == T_CLASS && zend_string_equals_literal(new_name, "strict")) {
```

Comment:
@TysonAndre How do we reach this code? Something like:

```php
class strict
{
    
}

$newClass = new strict();
```
?

Reply:
The intent of this check was to resolve `use strict;`

```
» php test.php
Warning: The use statement with non-compound name 'strict' has no effect in /path/to/test.php on line 2
» cat test.php 
<?php
use strict;
```

Reply:
In case someone writing PHP mixed it up with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode